integration-cli/docker_api_logs_test.go

```diff
@@ -82,3 +86,96 @@ func (s *DockerSuite) TestLogsAPIContainerNotFound(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusNotFound)
 }
+
+func (s *DockerSuite) TestLogsAPIUntilFutureFollow(c *check.C) {
```

Comment:
I'm wondering if this test is really valid as it is.
I think we need to read all the logs collected and verify that it indeed stops when expected.